tests/test_tweet_layout.py

```
@@ -291,3 +292,164 @@ def test_build_nodes_for_uin_strips_source_instance_links():
     text = "".join(getattr(c, "text", "") for c in node_content)
     assert "https://nitter.example.com" not in text
     assert "https://example.com/page" in text
+
+
+def _img_media(path="/tmp/a.jpg"):
```

Ruff (0.16.3)

[error] Probable insecure usage of temporary file or directory: "/tmp/a.jpg"

(S108)